src/commands/PublishCommand.js

```diff
@@ -292,7 +292,7 @@ export default class PublishCommand extends Command {
       const version = this.updatesVersions[depName];
 
       if (deps[depName] && version) {
-        deps[depName] = "^" + version;
+        deps[depName] = this.flags.exact ? version : "^" + version;
```

Comment:
It's not obvious to me just looking at this that it works.

I mean, I know it _does_ (thanks to your tests), but I'm not confident about operator precedence between `?:` and `+`.  This would be a clearer for me with a set of parentheses.